business_requirement_deliverable_sale/models/sale_order.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business requirement',
+    )
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    business_requirement_deliverable_section_id = fields.Many2one(
```

Comment:
No need of this. What you have to do is to create sales sections, and link them with BR section, and also put the sale order lines with the corresponding sale sections.